mailpoet/lib/Automation/Engine/Builder/UpdateAutomationController.php

```diff
@@ -141,4 +155,22 @@ private function stepChanged(Step $a, Step $b): bool {
     unset($bData['args']);
     return $aData === $bData;
   }
+
+  private function unscheduleAutomationRuns(Automation $automation): void {
+    $runIds = [];
+    $runs = $this->automationRunStorage->getAutomationRunsForAutomation($automation);
+    foreach ($runs as $run) {
+      $runIds[$run->getId()] = $run;
+    }
+
+    $actions = $this->actionScheduler->getScheduledActions(['hook' => Hooks::AUTOMATION_STEP, 'status' => 'pending']);
```

Comment:
There is also a constant `ActionScheduler_Store::STATUS_PENDING`, but that's very minor (and not needed to change).

Reply:
Thank you for pointing that out. I relied on the copilot and its suggested code without looking if there was a better way.